### ISSUE TITLE
Greyed images in dpretrained rendering

### ISSUE THREAD

**Issue by yaseryacoob** [zhengqili/Neural-Scene-Flow-Fields#13]
I followed the instruction on the kid's data and I am getting this kind of image 
![YYtemp](https://user-images.githubusercontent.com/8571131/115404569-73e0ca00-a1bb-11eb-95e8-65cd038ed483.png)

The prep commands all worked fine, and the motion mask looks alright. I redid the whole process twice but no change.
It is on linux/cuda11,pytorch 1.8

Any insight as to why?

**Comment by laomao0** [comment]
I obtain similar results. My env is python36, torch  16.
I do not know where is the bug.

**Comment by laomao0** [comment]
I think you do not load the trained weights correctly.

**Comment by laomao0** [comment]
> I think you do not load the trained weights correctly.

check render_utils.py --> print('LOADING SF MODEL!!!!')

**Comment by yaseryacoob** [comment]
Good direction, you instructions said to have the chpt at nsff_exp/logs/kid-running_ndc_5f_sv_of_sm_unify3_F00-30/360000.tar  but the render_utils.py  looks in nsff_exp/logskid-running_ndc_5f_sv_of_sm_unify3_testing_F00-30 for the tar file

once I copied where it was looking it worked fine.

Thanks!
